app/pb_kits/playbook/pb_legend/_legend.html.erb

```diff
@@ -5,7 +5,7 @@
     class: object.classname) do %>
   <%= pb_rails("body", props: {color: object.body_color}) do %>
     <span class="pb_legend_indicator_circle"></span>
-    <%= pb_rails("title", props: { dark: object.dark, text: object.prefix_text, tag: "span", size: 4 }) %>
+    <%= pb_rails("title", props: { text: object.prefix_text, tag: "span", size: 4 }) %>
```

Comment:
Can we just pass the dark prop to the title kit instead?

Reply:
This seems fine as well.